sdcm/cluster.py

```diff
@@ -993,6 +993,13 @@ def _get_core_pids(self):
         #     Tue 2020-01-14 16:18:56 +07    6321  1000  1000   3 present   /usr/bin/scylla
         #     Tue 2020-01-14 16:31:39 +07   18554  1000  1000   3 present   /usr/bin/scylla
         for line in result.splitlines():
+            if "bash" in line:
```

Comment:
isn't it too general? just "bash" would be enough?

Reply:
the only string in those line, is the name of the process.
we mostly care about scylla coredumps, and it won't filter those

Reply:
Can we match the full path of bash?

'/usr/bin/bash' in line

```
$ sudo coredumpctl --no-pager --no-legend
Fri 2020-07-10 01:11:16 UTC    1796  1004  1005  11 * /usr/bin/bash
```